rimage/depth_map.go

```diff
@@ -117,28 +116,21 @@ func (dm *DepthMap) SubImage(r image.Rectangle) *DepthMap {
 // or if it can be converted into one.
 func ConvertImageToDepthMap(img image.Image) (*DepthMap, error) {
 	switch ii := img.(type) {
-	case *LazyEncodedImage:
-		decoded, err := decodeImage(
-			context.Background(), ii.RawData(), ii.MIMEType(), ii.Bounds().Dx(), ii.Bounds().Dy(), false)
-		if err != nil {
-			return nil, err
-		}
-		return ConvertImageToDepthMap(decoded)
 	case *DepthMap:
 		return ii, nil
 	case *imageWithDepth:
 		return ii.Depth, nil
 	case *image.Gray16:
 		return gray16ToDepthMap(ii), nil
-	case *image.NRGBA64:
-		return nrgba64ToDepthMap(ii), nil
+	case *image.NRGBA:
```

Comment:
From what I understand, we will be going back to using a raw Depth mime type as well. We were using NRGBA64 for both raw Depth and raw Color (since depth needs 16bit pixels, we changed the color channels to also use 16 bit). 
Switching back to NRGBA means color is 8-bit again (which is more standard, and it will be easier for users to understand the custom rgba mime type)

In another PR, we will being back the `.dat` files and the custom Depth mime type. But that isn't necessary in this PR, as we pretty much just use PNG for most of our depth images.